plugins/ltac/rewrite.ml

```diff
@@ -416,11 +426,18 @@ module TypeGlobal = struct
 
 
   let inverse env (evd,cstrs) car rel =
+    let evd, car = Evarsolve.refresh_universes ~onlyalg:true (Some false) env evd car in
```

Comment:
Why status:rigid here?

Reply:
I refactored so that everything uses the same refresh_universes with a flexible status

Reply:
this one is still rigid isn't it?

Reply:
It's now flexible.